openvdb/openvdb/python/pyproject.toml

```diff
@@ -0,0 +1,36 @@
+[build-system]
+requires = [
+    "scikit_build_core",
```

Comment:
`scikit_build_core` is not yet stable, I would recommend pinning the minor version until it becomes stable.

Reply:
Agreed - doing this now.